docs/src/pages/demos/text-fields/TextFields.js

```diff
@@ -92,7 +92,7 @@ class TextFields extends React.Component {
           id="password"
           label="Password"
           className={classes.textField}
-          type="password"
+          variant="password"
```

Comment:
This is breaking the password field = password is shown
https://github.com/mui-org/material-ui/pull/10313